pkg/action/cl001/ac000/plan.go

```diff
@@ -16,17 +16,17 @@ var Plan = []plan.Step{
 	},
 	{
 		Action:  "ac002",
-		Backoff: plan.NewBackoff(24*time.Minute, 3*time.Minute),
+		Backoff: plan.NewBackoff(30*time.Minute, 3*time.Minute),
```

Comment:
For our cluster transition metrics we have a threshold. This threshold should be aligned with this number. Did we say it is 30 minutes?

Reply:
I noticed when we run cl001 ac002 it takes a bit longer due to the fact we creating 3 master nodes, it happened to me that it was waiting for the last master node a longer (2/3 healthy).

Reply:
In our cluster transition metrics we allow the cluster to create within 30 minutes.

Reply:
Nice, then this aligns. 🚀